packages/x-engine/readme.md

```diff
@@ -19,35 +19,41 @@ You'll also need to install your chosen runtime and any related dependencies. So
 - [Preact](https://preactjs.com/)
 - [Rax](https://alibaba.github.io/rax/)
 - [React](https://reactjs.org/)
-- [VHTML](https://github.com/developit/vhtml)<sup>†</sup>
+- [vdo](https://github.com/DylanPiercey/vdo)
 
 \* Usage of Hyperapp depends on a small modification to higher-order components to accept `children` as a second argument rather than receiving them appended to `props`.
 
-† The current release of the VHTML module has compatibility issues and is therefore not viable for production use without modification.
-
 ## Configuration
 
-To start you must specify your runtime configuration within `package.json`. This instructs `x-engine` which modules to load for both the server and for the browser environments.
+To start you must specify your runtime configuration within `package.json`. This instructs `x-engine` which modules to load for your environment. You may specify different runtimes for server and browser rendering depending on your needs.
 
-You only need to specify the environments you need and you may specify different runtimes depending on your needs.
+This module includes several presets for popular tools and frameworks. For example to use Hyperons on the server and Preact in the browser very little configuration is required:
```

Comment:
maybe list what presets are available? "very little configuration" also feels a little presumptious, i'd suggest something along the lines of "all the configuration you need is"